Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -629,16 +627,18 @@ ErrorOr<void> ChessWidget::export_pgn(Core::File& file) const
     // Tag Pair Section
     TRY(file.write_until_depleted("[Event \"Casual Game\"]\n"sv.bytes()));
     TRY(file.write_until_depleted("[Site \"SerenityOS Chess\"]\n"sv.bytes()));
-    TRY(file.write_until_depleted(DeprecatedString::formatted("[Date \"{}\"]\n", Core::DateTime::now().to_deprecated_string("%Y.%m.%d"sv)).bytes()));
+    TRY(file.write_until_depleted(String::formatted("[Date \"{}\"]\n", Core::DateTime::now().to_deprecated_string("%Y.%m.%d"sv)).release_value_but_fixme_should_propagate_errors().bytes()));
```

Comment:
You can use `write_formatted()` here.
And that's a fallible context, so no need to `release_value_but_fixme_should_propagate_errors()`, you can use `TRY()`